src/states/node.rs

```diff
@@ -1418,13 +1418,14 @@ impl Node {
     }
 
     fn send_candidate_approval(&mut self) {
-        let (new_id, client_auth) = match self.peer_mgr.verified_candidate_info(&self.log_ident()) {
-            Err(_) => {
-                trace!("{} No candidate for which to send CandidateApproval.", self);
-                return;
-            }
-            Ok(result) => result,
-        };
+        let (new_id, client_auth, old_public_id) =
```

Comment:
Either rename `new_id` to `new_public_id` or `old_public_id` to `old_id`, to keep the names consistent.